.gitignore

```
@@ -43,4 +43,6 @@ local.properties
 buck-out/
 \.buckd/
 *.keystore
-      
\ No newline at end of file
+      
+# Web build
+web/
```

Se comment in `package.json` for the reasoning.

```suggestion
lib/

```